tests/stress/stress_test.go

```diff
@@ -158,3 +188,11 @@ func Benchmark25Environments25Deploys25Services25LoadBalancers(b *testing.B) {
 		NumServices:      25,
```

Comment:
By themselves, these metrics don't tell me much. If/when the API starts crapping out, its hard to tell what the cause may be. I think we need to have more logical groupings of tests for each entity, probably a small/med/large size for that entity type with all other variables as consistent as possible. 

```
Benchmark1Family10Deploys(b *testing.B)
Benchmark1Family50Deploys(b *testing.B)
Benchmark1Family100Deploys(b *testing.B)

Benchmark10Families10Deploys(b *testing.B)
Benchmark50Families50Deploys(b *testing.B)
Benchmark100Families100Deploys(b *testing.B)

Benchmark5Environment(b *testing.B)
Benchmark10Environments(b *testing.B)
Benchmark25Environments(b *testing.B)

Benchmark1Environment5Services(b *testing.B)
Benchmark1Environment25Services(b *testing.B)
Benchmark1Environment50Services(b *testing.B)

Benchmark10Environments25Services(b *testing.B)
Benchmark25Environments50Services(b *testing.B)

Benchmark1Environment5LoadBalancers(b *testing.B)
Benchmark1Environment25LoadBalancers(b *testing.B)
Benchmark1Environment50LoadBalancers(b *testing.B)

Benchmark10Environments25LoadBalancers(b *testing.B)
Benchmark25Environments50LoadBalancers(b *testing.B)

Benchmark1Environment10Tasks(b *testing.B)
Benchmark1Environment50Tasks(b *testing.B)
Benchmark1Environment100Tasks(b *testing.B)

Benchmark10Environments50Tasks(b *testing.B)
Benchmark25Environments100Tasks(b *testing.B)

// with each individual entity tested, now we can test the aggregate scenarios
Benchmark1Environment5Services5LoadBalancers10Tasks(b *testing.B)
Benchmark10Environments25Services25LoadBalancers50Tasks(b *testing.B)
Benchmark25Environments50Services50LoadBalancers100Tasks(b *testing.B)
```

Reply:
With this in mind, I think it could be a decent idea to keep `const` values for the `min`, `max`, and `average` scale for each entity.

```
const (
    ENVIRONMENT_SCALE_MIN = 1
    ENVIRONMENT_SCALE_AVG = 10
    ENVIRONMENT_SCALE_MAX = 25
    SERVICE_SCALE_MIN     = 5
    SERVICE_SCALE_AVG     = 25
    SERVICE_SCALE_MAX     = 50
)

func BenchmarkMinEnvironmentsMinServices(b *testing.B)
func BenchmarkMinEnvironmentsAvgServices(b *testing.B)
func BenchmarkMinEnvironmentsMaxServices(b *testing.B)

func BenchmarkAvgEnvironmentsAvgServices(b *testing.B)
func BenchmarkMaxEnvironmentsMaxServices(b *testing.B)
```